### ISSUE TITLE
Title of games are not drawn properly on the menu screen

### ISSUE THREAD

**Issue by Instandhaltung** [Robz8/DSiMenuPlusPlus#154]
### Your console

Tell us which console do you have:

- [ ]  DS Phat
- [ ]  DS Lite
- [X]  DSi
- [ ]  DSi XL
- [ ]  Old 3DS
- [ ]  Old 3DS XL
- [ ]  Old 2DS
- [ ]  New 3DS
- [ ]  New 3DS XL
- [ ]  New 2DS XL

### CFW and configuration

Tell us which CFW do you have:

DSi:

- [X]  HiyaCFW
- [ ]  Not using CFW / DSiWarehax

### DSiMenu++ version

Tell us which versions you are using:

- [X]  Latest release _[v5.4.1]_
- [ ]  I've compiled myself _[insert here commit hash]_
- [ ]  Old release _[please specify version]_

### Expected behaviour

Text is seen as it should be

### Actual behaviour

Text in menu looks fuzzy and displays in system color (blue for me) or rarely in white (pictures below). Sometimes the player name on top screen is also affected, sometimes not.

### Steps to reproduce

I couldn't reproduce this every time, it seems a bit random but it usually happened when i...

1. reset the console by pressing the power button after having played a game
or
2. being in the options menu of DSiMenu++, saving the settings and going back to main menu.

![img_20180812_153959](https://user-images.githubusercontent.com/20743229/44005305-b79c20f0-9e71-11e8-9d5c-c75c833b9ca5.jpg)
![img_20180812_154749](https://user-images.githubusercontent.com/20743229/44005306-b7c2a4dc-9e71-11e8-9629-b96a04db4759.jpg)

**Comment by Robz8** [comment]
That's normal. Sometimes when graphics are being loaded, some of them overwrites the already loaded ones.

**Comment by piratesephiroth** [comment]
This used to happen kinda rarely before but it occurs way too often on the latest releases.

It's almost guaranteed to happen either as it starts or as we scroll through the games.

**Comment by Daeymon** [comment]
Agreed. Happening more often and to a more severe degree as well where the text can become unreadable.
